waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -131,7 +131,7 @@ waspNpmDeps =
       ("debug", "~2.6.9"),
       ("express", "~4.16.1"),
       ("morgan", "~1.9.1"),
-      ("@prisma/client", "2.22.1"),
+      ("@prisma/client", prismaVersion),
```

Comment:
Yep, I checked, but good to always double-check! :) The only other place we use `npx prisma` where we actually do not need it is in the Dockerfile, since it runs _after_ `npm install`